src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -170,117 +176,135 @@ def __init__(
+                            string_buffer.append("\n")
+                        except MetricError as ex:
+                            self.__logger.warning(
+                                "Failed to serialize metric. Skipping: %s", ex)
+
+                        if len(string_buffer) / 2 >= chunk_size:
+                            try:
+                                self._send_lines(string_buffer)
+                                string_buffer = []
+                            except Exception as ex:
+                                self.__logger.warning(
+                                    "Failed to export metrics: %s", ex)
+                                return MetricExportResult.FAILURE
+        try:
+            self._send_lines(string_buffer)
+        except Exception as ex:
+            self.__logger.warning(
+                "Failed to export metrics: %s", ex)
+            return MetricExportResult.FAILURE
         return MetricExportResult.SUCCESS
 
-    def _to_dynatrace_metric(self, metric: Metric):
+    def _send_lines(self, string_buffer):
+        serialized_records = "".join(string_buffer)
+        self.__logger.debug(
+            "sending lines:\n" + serialized_records)
+        with self._session.post(self._endpoint_url,
+                                data=serialized_records,
+                                headers=self._headers) as resp:
+            resp.raise_for_status()
+            self.__logger.debug(
+                "got response: {}".format(
+                    resp.content.decode("utf-8")))
+
+    def _sum_to_dynatrace_metric(self, metric: Metric, point: NumberDataPoint):
+        if isinstance(point.value, float):
+            return self._metric_factory.create_float_counter_delta(
+                metric.name,
+                point.value,
+                dict(point.attributes),
+                int(point.time_unix_nano / 1000000))
+        if isinstance(point.value, int):
+            return self._metric_factory.create_int_counter_delta(
+                metric.name,
+                point.value,
+                dict(point.attributes),
+                int(point.time_unix_nano / 1000000))
+
+    def _gauge_to_dynatrace_metric(self, metric: Metric,
+                                   point: NumberDataPoint):
+        if isinstance(point.value, float):
+            return self._metric_factory.create_float_gauge(
+                metric.name,
+                point.value,
+                dict(point.attributes),
+                int(point.time_unix_nano / 1000000))
+        if isinstance(point.value, int):
+            return self._metric_factory.create_int_gauge(
+                metric.name,
+                point.value,
+                dict(point.attributes),
+                int(point.time_unix_nano / 1000000))
+
+    def _histogram_to_dynatrace_metric(self, metric: Metric,
+                                       point: HistogramDataPoint):
+        return self._metric_factory.create_float_summary(
```

Comment:
Need to check whether its cumulative or delta. can also be done in a separate PR

Reply:
Now checking and dropping and logging cumulatives as of [2d01775](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/2d01775f272252daa002b671d1a760238bbcfc8c).